src/main/java/lutchat/Ui.java

```diff
@@ -0,0 +1,125 @@
+package lutchat;
+
+import java.util.Scanner;
+import java.util.ArrayList;
+
+/**
+ * The Ui class handles the user interface interactions for the application.
+ * It provides methods to display messages and prompt the user for input.
+ */
+public class Ui {
```

Comment:
I like that you have a dedicated `Ui` class for handling output from the chatbot. I left the handling of output to my main method which does make things a bit messy in Duke.java